arches/app/templates/views/components/file-workbench.htm

```diff
@@ -162,27 +176,32 @@ <h4 class="r-select-title" data-bind="text: $data.title"></h4>
                 <div class="file-select-window">
                    <div class="bord-top pad-ver file-select">
                        <div class="" style="padding: 5px">
-                           <h2>{% trans 'Upload Files' %}</h2>
-                           <h4>{% trans 'Drag & Drop your files onto this panel to upload' %}</h4>
-                       </div>
-                       <button type="button" class="btn btn-lg btn-file-select fileinput-button dz-clickable" data-bind="css: uniqueidClass">
-                           <i class="fa fa-file"></i>
-                           <span>{% trans "Select Files" %}</span>
-                       </button>
-                       <br>
-                       <div style="padding: 10px">{% trans "Adding files to this record is " %}
-                           <span><strong>{% trans "optional." %}</strong></span>
-                           <br>
-                           {% trans "You may upload as many files as you wish; check with the site admin on the maximum file size." %}
-                       </div>
+                            <h2>
+                                <span data-bind="text: translations.uploadFiles"></span>
+                            </h2>
+                            <h4>
+                                <span data-bind="text: translations.dropzone-file-upload-caption"></span>
```

Comment:
calls to `translations` should be camel cased

Reply:
Thanks.